spec/factories/ext_management_system.rb

```diff
@@ -89,13 +89,16 @@
   end
 
   factory :ems_openstack, :aliases => ["manageiq/providers/openstack/cloud_manager"], :class => "ManageIQ::Providers::Openstack::CloudManager", :parent => :ems_cloud do
+    after :build do |x|
+      x.authentications << FactoryGirl.create(:authentication, :userid => "admin", :password => "123456789")
+      x.authentications << FactoryGirl.create(:authentication, :userid => "user", :password => "abcdefgh",
+                                              :authtype => "amqp")
```

Comment:
I guess it could stay in the factory :ems_openstack_with_authentication, so it's consistent with the other factories. Maybe there is a good reason for moving it? :-)